packages/metro/src/ModuleGraph/worker/collectDependencies.js

```diff
@@ -251,6 +271,70 @@ function collectDependencies<TSplitCondition = void>(
   };
 }
 
+/** Extract args passed to `require.context` method. Exposed for testing. */
```

Comment:
> Exposed for testing

We don't seem to be using that here (and I don't think we should - better to write full test cases against `collectDependencies`)